sprokit/src/sprokit/pipeline_util/pipe_declaration_types.h

```diff
@@ -36,7 +36,6 @@
 
 #include <sprokit/pipeline/process.h>
 
-#include <boost/optional.hpp>
 #include <boost/variant.hpp>
```

Comment:
Completely unrelated to this branch, but it would be great to tackle boost/variant as well to get boost out of this header.

Reply:
That's on my list to do, but it is all wrapped up in the boost::visitor. Just need to settle on a visitor approach.